src/hooks/useMediaQuery/README.md

```diff
@@ -21,12 +21,13 @@ Because Orbit aims to be mobile-first, you should consider your base return with
 
 ## Return
 
-The `useMediaQuery` hook returns an object with boolean values depending on which viewport the user is currently on.
-
-| Name           | Type      | Description                                        |
-| :------------- | :-------- | :------------------------------------------------- |
-| isMediumMobile | `boolean` | `true` if viewport equals or is more than `414px`  |
-| isLargeMobile  | `boolean` | `true` if viewport equals or is more than `576px`  |
-| isTablet       | `boolean` | `true` if viewport equals or is more than `768px`  |
-| isDesktop      | `boolean` | `true` if viewport equals or is more than `992px`  |
-| isLargeDesktop | `boolean` | `true` if viewport equals or is more than `1200px` |
+The `useMediaQuery` hook returns an object with boolean values depending on which viewport the user is currently on or what system preferences have the device.
+
+| Name                 | Type      | Description                                        |
+| :------------------- | :-------- | :------------------------------------------------- |
+| isMediumMobile       | `boolean` | `true` if viewport equals or is more than `414px`  |
+| isLargeMobile        | `boolean` | `true` if viewport equals or is more than `576px`  |
+| isTablet             | `boolean` | `true` if viewport equals or is more than `768px`  |
+| isDesktop            | `boolean` | `true` if viewport equals or is more than `992px`  |
+| isLargeDesktop       | `boolean` | `true` if viewport equals or is more than `1200px` |
+| prefersReducedMotion | `boolean` | `true` if device uses reduced motion               |
```

Comment:
maybe add a comment about IE11, that it always defaults to `false` and then it's absolutely clear what's going on.